server/channels/db/migrations/postgres/000130_add_property_fields.up.sql

```diff
@@ -0,0 +1,34 @@
+DO
+$$
+BEGIN
+  IF NOT EXISTS (SELECT * FROM pg_type typ
+                            INNER JOIN pg_namespace nsp ON nsp.oid = typ.typnamespace
+                        WHERE nsp.nspname = current_schema()
+                            AND typ.typname = 'property_field_type') THEN
+    CREATE TYPE property_field_type AS ENUM (
```

Comment:
Have we considered how plugins could extend the property field types in the future? I'm afraid that enforcing this as a DB level might prevent us to allow plugins to register their own "property field types" on the fly.

Reply:
Sounds good to me, just wanted to make sure it's been considered! Resolving this